src/mmw/js/src/core/views.js

```diff
@@ -347,18 +348,28 @@ var ShareModal = BaseModal.extend({
 
     ui: {
         'signin': '.signin',
-        'copy': '.copy',
+        'copy': '#copy-link-button',
```

Comment:
I was experimenting with ZeroClipboard for a while and needed to individually target it. Since the template didn't have a `copy` class on the button, I went ahead and gave it an ID. But should've switched back before committing, to minimize number of changes required to introduce this functionality. Reverted now. Thanks!